weave

```diff
@@ -328,15 +300,82 @@ fi
 # main (remote and --local) - settings
 ######################################################################
 
+## Versions ##
+SCRIPT_VERSION="unreleased"
+IMAGE_VERSION=latest
+[ "$SCRIPT_VERSION" = "unreleased" ] || IMAGE_VERSION=$SCRIPT_VERSION
+IMAGE_VERSION=${WEAVE_VERSION:-$IMAGE_VERSION}
+
+## Image Names ##
+
+DOCKERHUB_USER=${DOCKERHUB_USER:-weaveworks}
```

Comment:
You cannot move this after the first use (which is in the call to `exec_remote` on line 260)